test/src/e2e_vm_tests/test_programs/should_fail/raw_ptr/raw_ptr_ret/src/main.sw

```diff
@@ -0,0 +1,6 @@
+script;
+
+fn main() -> raw_ptr {
```

Comment:
I would be nice to also have a positive test here.. even something simple, like a unit test, where you convert a `raw_ptr` to a `u64`, do some math, and then back, just to show that the type is functional.